tests/02_00_02-2wp_segwit_compatible.js

```diff
@@ -1,74 +1,58 @@
 const expect = require('chai').expect
-var { sequentialPromise, wait } = require('../lib/utils');
-const bitcoin = require('peglib').bitcoin;
-const rsk = require('peglib').rsk;
-const pegUtils = require('peglib').pegUtils;
-const pegAssertions = require('../lib/assertions/2wp');
-const rskUtilsLegacy = require('../lib/rsk-utils-legacy');
-const CustomError = require('../lib/CustomError');
 const rskUtils = require('../lib/rsk-utils');
 const { getRskTransactionHelpers } = require('../lib/rsk-tx-helper-provider');
+const { sendPegin, MINIMUM_PEGIN_VALUE_IN_BTC, ensurePeginIsRegistered } = require('../lib/2wp-utils');
+const { getBridge, getLatestActiveForkName } = require('../lib/precompiled-abi-forks-util');
+const { getBtcClient } = require('../lib/btc-client-provider');
+const { getDerivedRSKAddressInformation } = require('@rsksmart/btc-rsk-derivation');
+const btcEthUnitConverter = require('btc-eth-unit-converter');
+
 
-var federationAddress;
-var btcClient;
-var rskClient, rskClients;
-var pegClient;
-var test;
 let rskTxHelpers;
+let rskTxHelper;
+let btcTxHelper;
+
 
-const NETWORK = bitcoin.networks.testnet;
+const fulfillRequirementsToRunAsSingleTestFile = async () => {
+    await rskUtils.activateFork(Runners.common.forks.fingerroot500);
```

Comment:
Tests that start with `02_` are supposed to be the actual integration tests, the previous ones (starting with `01_` are supposed to be the regression tests. Although it is not exactly like that, it is the idea.

At this point, we want the latest fork to be active. The latest fork is now fingerroot, but when that changes we'll have to update all the tests. To avoid this, I suggest you change this to activate the latest existing fork

Reply:
For this I would suggest adding the following function to the `precompiled-abi-forks-util.js` file:

```js
/**
 * 
 * @returns {string} The name of the latest fork
 *
 */
const getLatestForkName = () => {
    const forks = Object.values(Runners.common.forks);
    return forks[forks.length - 1].name;
};
```

This way we always get the latest declared fork in `Runners.common.forks)`.

What do you think @marcos-iov ?

Reply:
Not sure if `precompiled-abi-forks-util.js` is the best place to have the function `getLatestForkName`. It is not really related to the precompiled.

Maybe we could have that function in `rsk-utils`?

Reply:
added to `rsk-utils`